pkg/mdns/controller.go

```diff
@@ -53,7 +53,7 @@ func (c *MicroShiftmDNSController) Run(ctx context.Context, ready chan<- struct{
 			strings.HasPrefix(name, "cni") {
 			continue
 		}
-		klog.Infof("starting mDNS server", "interface", name, "NodeIP", c.NodeIP, "Node", c.NodeName)
+		klog.Infof("starting mDNS server on interface %q, NodeIP %q, NodeName %q", name, c.NodeIP, c.NodeName)
```

Comment:
Should we capitalize the first word (Starting) to make is consistent with other messages?

Reply:
Done.